evergreen/create-expansions.sh

```diff
@@ -0,0 +1,118 @@
+export CARGO_NET_GIT_FETCH_WITH_CLI=true
+export INSTALLED_ODBC_PATH="$PWD/installed_odbc/install"
+export LD_LIBRARY_PATH="$INSTALLED_ODBC_PATH/lib"
+if [ "Windows_NT" == "$OS" ]; then
+    export PATH="$HOME/.rustup/bin:$HOME/.cargo/bin:$LD_LIBRARY_PATH:$PATH"
+else
+    export PATH="$HOME/.cargo/bin:$LD_LIBRARY_PATH:$PATH"
+fi
+export DUMP_FOLDER=dumps
+export LOCAL_DUMP_ORIGINAL_REG_VAL=local_dump_original_value.reg
+export MONGOODBC_DEBUGGING_INFO_ARCHIVE=crashDebuggingInfo
+export SCRIPT_FOLDER=resources
+export COMPLIANCE_REPORT_NAME="mongo-odbc-driver_compliance_report.md"
+export STATIC_CODE_ANALYSIS_NAME="mongo-odbc-driver.sast.sarif"
+export FEATURE_FLAGS=""
+export PRODUCT_NAME="mongoodbc"
+export PATH_PREFIX=""
```

Comment:
This could have been added for the S3 path.  I don't see it being used, it can be removed if not.

Reply:
Yeah I removed it since the tasks have to be duplicated :(